2021-02-05 20:44:05,244:WARNING:discord.client: PyNaCl is not installed, voice will NOT be supported
2021-02-05 20:44:05,270:INFO:discord.client: logging in using static token
2021-02-05 20:44:06,717:INFO:discord.gateway: Shard ID None has sent the IDENTIFY payload.
2021-02-05 20:44:07,331:INFO:discord.gateway: Shard ID None has connected to Gateway: ["gateway-prd-main-bdsz",{"micros":77672,"calls":["discord-sessions-prd-2-92",{"micros":76264,"calls":["start_session",{"micros":31235,"calls":["api-prd-main-npxp",{"micros":28291,"calls":["get_user",{"micros":2136},"add_authorized_ip",{"micros":1725},"get_guilds",{"micros":6916},"coros_wait",{"micros":1}]}]},"guilds_connect",{"micros":1,"calls":[]},"presence_connect",{"micros":19526,"calls":[]}]}]}] (Session ID: 06b303e6c5c42e2d07524d80d4b45842).
2021-02-05 20:44:14,090:ERROR:discord: No unique_id is received from the command
Traceback (most recent call last):
  File "/home/prachipathrayan/Sites/discord-bot/services/content.py", line 210, in mark_ques_status
    unique_id=command.content.split(' ')[1]
IndexError: list index out of range
2021-02-05 20:45:44,382:INFO:discord.client: Cleaning up tasks.
2021-02-05 20:45:44,383:INFO:discord.client: Cleaning up after 1 tasks.
2021-02-05 20:45:44,383:INFO:discord.client: All tasks finished cancelling.
2021-02-05 20:45:44,384:INFO:discord.client: Closing the event loop.
